### DISCUSSION TITLE
PrintToPDF from CLI

### DISCUSSION THREAD

**Discussion by ghost** [sumatrapdfreader/sumatrapdf#4502] [Q&A]
Hello,
please is it possible to print to PDF from command line? I can not find this information anywhere.
Thank you in advance.
Petr

**Comment by GitHubRulesOK** [comment]
Not really necessary but is there. in pre-release it is in help 
or on web site https://www.sumatrapdfreader.org/docs/Command-line-arguments

Printing to PDF is in Windows controlled by "Microsoft Print to PDF" driver so either you set the default printer to Microsoft PDF and command line print to default 
`-print-to-default` : prints all files indicated on this command line to the system default printer. 

OR you have a different default and you
`-print-to <printer-name>` : prints all files indicated on this command line to the named printer. `-print-to "Microsoft Print to PDF"`

 but either way that printer quality will usually be poor compared to any other virtual PDF printer

**Reply by ghost** [reply]
Thank you! I like Sumatra but I probably should not push Sumatra somewhere where it is not intended for. Have a nice day.
